src/herder/UpgradesTests.cpp

```diff
@@ -210,21 +211,19 @@ testListUpgrades(VirtualClock::time_point preferredUpgradeDatetime,
                  bool shouldListAny)
 {
     auto cfg = getTestConfig();
-    cfg.LEDGER_PROTOCOL_VERSION = 10;
     cfg.TESTING_UPGRADE_DESIRED_FEE = 100;
     cfg.TESTING_UPGRADE_MAX_TX_PER_LEDGER = 50;
     cfg.TESTING_UPGRADE_RESERVE = 100000000;
     cfg.TESTING_UPGRADE_DATETIME = preferredUpgradeDatetime;
 
     auto header = LedgerHeader{};
-    header.ledgerVersion = cfg.LEDGER_PROTOCOL_VERSION;
+    header.ledgerVersion = 10;
```

Comment:
wondering about the value `10`, either use `MAX_SUPPORTED_LEDGER_PROTOCOL_VERSION + 1` (if you want a version not supported - you can define a new constant that can be used in tests) or use `MAX_SUPPORTED_LEDGER_PROTOCOL_VERSION`